about.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="responsive-style.css">
+  <script src="mobileMenu.js" defer></script>
+</head>
+<body>
+  <section id="home">
+    <nav>
+      <a href="index.html"><img src="img/logo.png" alt="" class="cclogo"></a>
+      <img src="img/mobile_menu.png" alt="" class="hamburger">
+      <ul class="link-items">
+        <li><a href="about.html">About</a> <img src="img/close_mobile_menu.png" alt=""></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome-about">
+      <p class="hello hello-about">"Hello! Sharing World"</p>
+      <h1 class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <P class="intro-about">The Global Summit brings together the community of  
+        experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+        This year, we hope to expand our invitation list including organizations and individuals.</P>
+        <div class="please-contact-us">
+          <p class="contact-msg">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+          <p class="contact-email">ccsummit2015@cckorea.org</p>
```

Comment:
- [ ] As I said the text on the home page, please update this one too to match the font weight of the original design on the left.
- [ ] The paragraph text is not using the same color, please update it

<img width="1046" alt="4" src="https://user-images.githubusercontent.com/77843050/211073561-54829727-08f0-454e-9150-0ccd90c7243b.png">